cs4360/libs/crm/nest/dao/src/lib/entities/contact.entity.ts

```diff
@@ -23,19 +24,21 @@ export class ContactEntity {
   lastName: string;
 
   @Column('varchar', { length: 50, nullable: true})
-  company: string
+  company?: string
 
   @Column('varchar', {length: 250, nullable: true})
-  notes: string
+  notes?: string
 
-  @OneToMany(type => EmailEntity, email => email.contact, {
+  @ManyToMany(type => EmailEntity, email => email.contact, {
     cascade: true
   })
+  @JoinTable()
   emails?: EmailEntity[];
```

Comment:
There was no join table being created yet, now we've got one ^.^

Reply:
awesome!